### PR TITLE
Add themes and threshold to ProgressBar

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
* Added `appearance` prop with `theme` and `size` to ProgressBar
* Added threshold to ProgressBar

FireShot Capture 163 - Colony - localhost

To test it:

1. Copy and paste this whenever you want(ProgressBar needs to be imported of course): 
`<ProgressBar value={50} appearance={{ theme: 'dark', size: 'small' }} threshold={70} />`.
2. ?????
3. Profit

Resolves DEV-238

### CODE REVIEW

**Review by alicjakujawa — APPROVED** [review]
Almost there 🔥 
Small nitpicks:
1. Add border-radius: 1px to threshold
2. Change bg color to 254,94,124,1

Pre-approving this

**Review by rdig — CHANGES_REQUESTED** [review]
This is all good, except for the threshold line not having the proper size when `size` is set to `normal`

File: src/modules/core/components/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import { defineMessages, useIntl } from 'react-intl';
 
+import { getMainClasses } from '~utils/css';
+
 import styles from './ProgressBar.css';
 
+interface Appearance {
+  theme?: 'default' | 'dark';
```
Comment (rdig) [inline src/modules/core/components/ProgressBar/ProgressBar.tsx:9]: So the 'default' theme, when in 'normal' size, has a background wrapper:

Screenshot from 2021-03-12 11-41-19

While, in the 'small' size, it does not

Screenshot from 2021-03-12 11-41-14

It think we should remove it for the 'normal' size as well to keep it consistent (an just have the grey "fill" portion of the bar)

**Review by rdig — APPROVED** [review]
I'm good with this now, however, please address the comment that I left before about the default small vs. normal styles.

Screenshot from 2021-03-12 11-41-24

File: src/modules/core/components/ProgressBar/ProgressBar.tsx (hunk 1 above)
Comment (ArmandoGraterol) [reply src/modules/core/components/ProgressBar/ProgressBar.tsx:9]: Sounds good to me! It also looks better without the wrapper

**Pull request merged**